styles/variables-base.less

```diff
@@ -524,25 +524,25 @@
 @agate-mediacontrols-button-bg-height: inherit;
 @agate-mediacontrols-button-bg-width: inherit;
 @agate-mediacontrols-button-client-width: inherit;
-@agate-mediacontrols-button-height: inherit;
+@agate-mediacontrols-button-height: none;
 @agate-mediacontrols-button-icon-padding: inherit;
-@agate-mediacontrols-button-margin: inherit;
-@agate-mediacontrols-button-min-width: inherit;
-@agate-mediacontrols-icon-font-size: inherit;
+@agate-mediacontrols-button-margin: 0 30px 0 30px;
+@agate-mediacontrols-button-min-width: none;
+@agate-mediacontrols-icon-font-size: @agate-icon-font-size;
 @agate-mediacontrols-icon-margin: inherit;
 @agate-mediacontrols-icon-padding: inherit;
 @agate-mediacontrols-playpause-button-bg-border-radius: inherit;
 @agate-mediacontrols-playpause-button-bg-height: inherit;
 @agate-mediacontrols-playpause-button-bg-width: inherit;
 @agate-mediacontrols-playpause-button-client-width: inherit;
-@agate-mediacontrols-playpause-button-height: inherit;
+@agate-mediacontrols-playpause-button-height: @agate-icon-font-size * 1.1;
 @agate-mediacontrols-playpause-button-padding: inherit;
-@agate-mediacontrols-playpause-icon-font-size: inherit;
+@agate-mediacontrols-playpause-icon-font-size: @agate-icon-font-size;
```

Comment:
fun fact :) It seems the className `playPauseIcon` where this variable is called is not used in the component. We should check this.